src/components/LandingPage/ComponentTwo/hero/hero.css

```diff
@@ -1,79 +0,0 @@
-@import url("https://cdn.jsdelivr.net/npm/bootstrap-icons@1.6.0/font/bootstrap-icons.css");
-
-.jumbotron{
-    background:rgba(0, 0, 0, 0.5) url("../../../../Assets/images/divfest.JPG");
-    background-repeat: no-repeat;
-    background-size: cover;
-    height:135vh;
-    width: auto;
-    border-radius: 0px;
-    background-blend-mode: darken;
-}
-
-.jumbotron h1{
-    margin: 30px 0;
-    line-height:1.5;
-    font-weight: bold;
-}
- .text{
-    color: white;
- }
-.container{
-    padding-top: 380px;
-    
-    
-}
-@media only screen and (max-width: 1000px){
-    .container{
-        padding-top: 300px;
-    }
-    .jumbotron h1{
-        line-height:1.7;
-        font-size: 30px;
-    }
-    .jumbotron{
-        height:auto;
-        width: auto;
-    }
-}
-.bi-star-fill{
-    font-size: 2rem; 
-    color: white;
-}
-.bi-dash-lg{
-    font-size: 3rem;
-    vertical-align: -.200em;
-    border-radius: 0;
-}
-@media only screen and (max-width: 760px){
-    .container{
-        padding-top: 250px;
-    }
-    .jumbotron h1{
-        line-height:1.7;
-        font-size: 30px;
-    }
-    .jumbotron{
-        height:auto;
-        width: auto;
-    }
-}
-
-@media only screen and (max-width: 480px){
-    .container{
-        padding-top: 180px;
-    
-    }
-    .jumbotron h1{
-        line-height:1.2;
-        font-size: 20px;
-    }
-    .jumbotron h4{
-        line-height: 0.5em;
-        font-size: 20px;
-    }
-    .jumbotron{
-        height:auto;
-        width: auto;
-    }
-}
```

Comment:
Why was this file deleted?

Reply:
It was not deleted. I just renamed it so that it acts as a module.